mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -71,38 +72,41 @@ def __remote_init(self):
         # Automatic selection and allocation of a resource
         try:
             self.selected_resource = self.client.allocate(self.allocate_requirements)
-
-        except self.remote_module.resources.ResourceError as e:
-            self.logger.prn_err("can't allocate resource: '%s', reason: %s"% (self.platform_name, str(e)))
+        except Exception as error:
```

Comment:
Typically we don't want to do general "catch all" exceptions. Is there not a more specific exception you can "except" on?

Reply:
I totally agree that generic catch should be avoided. When I did these changes I was thinking same thing and how to tackle case when exception class name is client specific. Maybe only way to avoid generic catch is to specify exception class name to genric and document it.. that would be breaking change..

Reply:
Actually I think current approach is just good in this case. All generic try-catch are very restricted so that they catch only client call exceptions. There is no any "wide" try-catches in this file. Also this way we doesn't need to document here specific exception class names - it's enough that client throws any exception which based on `Exception`. And bonus - doesn't need to change any client libraries..  ;) . @bridadan  what do you think ?

Reply:
I don't have a better idea so I suppose it's ok for now!